src/setupTests.ts

```diff
@@ -1,4 +1,22 @@
 import { configure } from 'enzyme'
 import Adapter from 'enzyme-adapter-react-16'
+import { JSDOM } from 'jsdom'
 
 configure({ adapter: new Adapter() })
+
+const { window } = new JSDOM('<!doctype html><html><body></body></html>')
```

Comment:
Let's use [`jsdom-global`](https://www.npmjs.com/package/jsdom-global) instead of setting up jsdom manaully?
```suggestion
import 'jsdom-global/register'
```